templates/base.html

```diff
@@ -46,7 +71,7 @@
           <span class="navbar-toggler-icon"></span>
         </button>
         <div class="collapse navbar-collapse" id="navbarText">
-          <ul class="navbar-nav mr-auto">
+          <ul class="navbar-nav me-auto">
```

Comment:
Could you tell me what does this difference affect?

Reply:
this is from the `ui-update` branch where I updated to using bootstrap 5.
the css classes navbar, btn and etc are from the bootsrap css that we are importing so when we upgraded to bs5
some classes names changed I have no idea why :)

Reply:
thanks